src/chaat/handler/handler.clj

```diff
@@ -0,0 +1,52 @@
+(ns chaat.handler.handler
+  (:require [ring.util.response :as res]
+            [chaat.model.user :as model.user]
+            [java-time.api :as jt]
+            [chaat.handler.validation :as handler.validation]
```

Comment:
[non-blocking, weak suggestion]
Not something that you need to address right now, but perhaps validations could be its own namespace, rather than nested under handlers?

Some options:
* `validation`—all validations go in a `validations.clj` file. This would contains validations needed across layers, like handler, model, business logic etc
* `validation.user`—each model has its own validation division, eg `validations/user.clj`

For simplicity, I might consider the former, since you have few models at the moment.

Reply:
There are multiple validations happening at the handler layer as well as the model layer.
Should we be combining the validations across layers and entities into a single file? (Even though it will be small after all this)

Reply:
Maybe an option three:
* `validation.model`, `validation.handler`, etc which will mimic the model-handler-db division of the application. Basically the inverse of what we are doing now.

My reasoning behind one file is that it's easy to change, and less baked in; we could defer the organisation of validations until more code is written and we have better information on which approach would work the best.

Reply:
Thanks for the suggestions, I'll start thinking about how to organize the validations and make this change in an upcoming PR.